### ISSUE TITLE
very slow opening file after updated atom

### ISSUE THREAD

**Issue by david43** [icetee/remote-ftp#1286]
### Prerequisites

* [x] Put an X between the brackets on this line if you have done all of the following:
    * Checked that your issue isn't already filed: https://github.com/icetee/remote-ftp/issues

### Description
Before atom update:
immediately open file
After atom update:
more than 30sec 

already try other ftp server/other pc/reinstall atom / remote-ftp
### Steps to Reproduce
https://cl.ly/5bba633f9459

### Versions

Atom    : 1.39.1
Electron: 3.1.10
Chrome  : 66.0.3359.181
Node    : 10.2.0

**Comment by Greex** [comment]
Same observation here. 
Already checked other FTP-Settings and Server-Settings. The issue startet with the atom update to 1.39.1

**Comment by icetee** [comment]
Are you using Linux?

https://github.com/atom/atom/releases/tag/v1.39.1

**Comment by icetee** [comment]
I was reproduced on windows

**Comment by Greex** [comment]
> Are you using Linux?

I'm running atom on Windows 10 - FTP-Servers are all Linux (Ubuntu/Debian).

**Comment by david43** [comment]
> Are you using Linux?
> 
> https://github.com/atom/atom/releases/tag/v1.39.1

windows 10 
I try reinstall old atom version.
It work perfectly

**Comment by icetee** [comment]
@david43 Which version are you on now? `v1.39.0`?

**Comment by david43** [comment]
> @david43 Which version are you on now? `v1.39.0`?

1.38.2

**Comment by beop07** [comment]
Same symptoms.
OS : windows 10
ATOM : 1.39.1
FTP Servers : Linux

VSCODE is being used temporarily because of this symptom.

Thank U.

**Comment by ghost** [comment]
Same here on macOS

**Comment by Aleksandr-Mukhanov** [comment]
On mac OS Mojave also opens files via FTP for a very long time - it is not possible to work! ((
ATOM : 1.39.1

**Comment by AlexNodex** [comment]
Same observation, the lag is due to downloading the file from FTP and not the opener itself.

MacOS Mojave, Latest Atom

**Comment by xembill** [comment]
same problem on Atom:V1.39.1
Windows 10
FTP Server : Linux

**Comment by kmartin26** [comment]
Same problem, started from Atom 1.39.0
OS: Windows 10

**Comment by sgmckay25** [comment]
For me same too after update to atom 1.39.1
Os Windows 10

**Comment by remcoov** [comment]
Same problem after updating to Atom 1.39.1
OS: MacOS Mojave

**Comment by janmarkuslanger** [comment]
Anything we can do to help with that bug?

**Comment by irpsv** [comment]
Resolve the situation: connect via SFTP.
But with FTP same problem.

Windows 10
Atom 1.39.1

**Comment by AlexNodex** [comment]
Just updated to Atom 1.4.0 on Mac and it's still the same.

It must be something that's changed since 1.3.8

**Comment by edgarwe** [comment]
If you can use SFTP it works perfectly, for me on Mac.
Just lost the logs in atom.

**Comment by ipportunities** [comment]
Has anyone found a solution for this problem?

I'm using version 1.38.2 now, that works fine. Problem starts with 1.39.0-beta2. 

I'm ftp'ing securely over port 21

**Comment by kjin1** [comment]
problem on Atom: v1.40.1
MacOS Mojave 10.14.6(18G87)
FTP Server : Linux

"SFTP" work perfectly same version

**Comment by kilazer** [comment]
A very long download opening FTP files.
Reinstalling the atom and the plugin did not help. In version atom 1.38.2, everything works well. The problem is in version 1.40.1.
ubuntu 19.04
atom 1.40.1 x 64

**Comment by kevinmu17** [comment]
Weird thing is, while having this issue in Atom I am exploring VScode with SFTP extension and i get the same results! Also slow when trying to open files just like in Atom

**Comment by ttschense** [comment]
same here. take ages to load a file from remote ftp. 
1.40.1 x64
macOS 10.13.6

works fine with atom 1.38.2 again (thanks for the info!).

**Comment by botondcs** [comment]
OS: Ubuntu 19.10
Atom: 1.40.1
FTP Server: Linux

Upload works instantly (fast as expected).

Download works fast too (watch the folders) but the closing of the transfer/connection takes ages. The file only gets shown once the transfer is done and the connection is closed.

**Comment by qbadev** [comment]
Same here:
Ubuntu 19.04
Atom 1.41.0
remote-ftp 2.2.2
Very long opening files
...
Also tried to use ftp-remote-edit plugin and it works fine as expected.
...
Downgrading to 1.38

**Comment by icetee** [comment]
I think resolved the new remote-ftp version (v2.2.3). Can you check?

**Comment by botondcs** [comment]
Works for me now.

remote-ftp: 2.2.3
OS: Ubuntu 19.10
Atom: 1.42.0

**Comment by qbadev** [comment]
I confirm: works fine now.
Edit: another issue came up - can't connect to SFTP servers. Had to get back to v2.2.2. Sorry
